scripts/core/device.yml

```
@@ -426,6 +428,8 @@ ordinal: "0"
 analogue:
     - "**clRetainDevice**"
 details:
+    - "Increments the device reference count if `hDevice` is a valid sub-device created by a call to `urDevicePartition`.
+    If `hDevice` is a root level device (e.g. obtained with `urDeviceGet`), the reference count remains unchanged."
```

This indentation looks a bit off

```suggestion
      If `hDevice` is a root level device (e.g. obtained with `urDeviceGet`), the reference count remains unchanged."
```

---

Fixed in the most recent commit